generator/examples/simple/cluster1/service2.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: config-service2
+    labels:
+        name: config-service2
+        version: cluster1
+    namespace: edge-namespace
+data:
+    conf.json: '{"processes":2,"threads":2,"endpoints":[{"name":"endpoint1","protocol":"http","execution_mode":"parallel","cpu_complexity":{"execution_time":"5s","method":"fibonacci","workers":2,"cpu_affinity":[22,23],"cpu_load":"100%"},"memory_complexity":{"execution_time":"5s","method":"swap","workers":24,"bytes_load":"100%"},"network_complexity":{"forward_requests":"asynchronous","response_payload_size":512,"called_services":[]}},{"name":"endpoint2","protocol":"http","execution_mode":"parallel","cpu_complexity":{"execution_time":"","method":"","workers":0,"cpu_affinity":null,"cpu_load":""},"memory_complexity":{"execution_time":"","method":"","workers":0,"bytes_load":""},"network_complexity":{"forward_requests":"asynchronous","response_payload_size":512,"called_services":[]}}]}'
```

Comment:
Not everyone has access to servers with 24 cores of CPU. I would suggest changing the values of cpu_affinity.